src/ansys/platform/instancemanagement/service.py

```diff
@@ -59,18 +60,23 @@ def _build_grpc_channel(
 
     @staticmethod
     def _from_pim_v1(service: ServiceV1):
-        """Build a Definition from the PIM API v1 protobuf object.
-
-        Args:
-            service (ServiceV1): raw PIM API v1 protobuf object
-
-        Raises:
-            ValueError: The raw protobuf message is not valid
-
-        Returns:
-            Definition: The PyPIM service
+        """Build a definition from the PIM API v1 protobuf object.
+
+        Parameters
+        ----------
+        service : ServiceV1
+            Raw PIM API v1 protobuf object.       
+
+        Returns
+        -------
+        type
```

Comment:
Should "type" be "string"? For PyAEDT, we used "type" as a default and then replaced it with an actual data type once we determined what it was.

Reply:
```suggestion
        Service
            The PyPIM service
```

I think I just messed up the type, it sould have been "Service"